lib/saml_authenticator.rb

```diff
@@ -357,6 +357,15 @@ def self.saml_base_url
 
   private
 
+  def idp_cert_multi
+    return unless setting(:cert_multi).present?
+
+    {
+      signing: [setting(:cert_multi)],
```

Comment:
You're only setting the secondary cert here, but [the README](https://github.com/SAML-Toolkits/ruby-saml#handling-multiple-idp-certificates) says `idp_cert` above will be ignored when this one is set:

> ## Handling Multiple IdP Certificates
> If the IdP metadata XML includes multiple certificates, you may specify the `idp_cert_multi`
parameter. When used, the `idp_cert` and `idp_cert_fingerprint` parameters are ignored.

~~There's [a specific test](https://github.com/onelogin/ruby-saml/blob/9efe57ae7614bcb7226986e52b6f0a3cf1ba4848/test/idp_metadata_parser_test.rb#L509) validating this behavior.~~ (that's an unrelated codepath we don't use)
[This](https://github.com/onelogin/ruby-saml/blob/9efe57ae7614bcb7226986e52b6f0a3cf1ba4848/lib/onelogin/ruby-saml/response.rb#L884) confirms they ignore `settings.get_idp_cert` when `settings.get_idp_cert_multi` returns non nil.

Thus I think this array should include both the cert given in the `idp_cert` and the `idp_cert_multi` site settings.

Reply:
Sounds good to me. I also think we should `.split("|")` the `setting(:cert_multi)` value so that people can supply multiple secondary certificates